src/IRremoteESP8266.h

```diff
@@ -179,6 +182,7 @@ enum decode_type_t {
 #define MITSUBISHI_MIN_REPEAT        1U  // Based on marcosamarinho's code.
 #define MITSUBISHI_AC_STATE_LENGTH  18U
 #define MITSUBISHI_AC_MIN_REPEAT     1U
+#define FUJITSU_AC_MIN_REPEAT        0U
```

Comment:
Nice! Now you've made me want to retro-fit that for all the other protocols. :-)